dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/redshift/JdbcPropBuilder.java

```diff
@@ -77,7 +75,11 @@ public JdbcPropBuilder prop(@Nonnull String prop, @Nonnull String val) {
     }
 
     private void addProp(String prop, String val) {
-        props.add(prop + punctuations.charAt(1) + URLEncoder.encode(val, UTF_8));
+        try {
+            props.add(prop + punctuations.charAt(1) + URLEncoder.encode(val, "UTF-8"));
```

Comment:
Do we prefer the string literal to the symbolic constant? Is this a JDK support/availability question? StandardCharsets is available on JDK7.

Reply:
It's the `URLEncoder.encode` overload which is missing in jdk 8.